src/index.js

```diff
@@ -4,34 +4,33 @@ import { Provider } from 'react-redux';
 import { createStore } from 'redux';
 import { BrowserRouter, Route, Switch } from 'react-router-dom';
 
-import './index.css';
-
-import Search from './components/Search.js';
-import Article from './components/Article.js';
-import Author from './components/Author.js';
-import Reader from './components/Reader.js';
-import Home from './components/Home.js';
+import './assets.js';
 
+import Search from './components/search';
+import Article from './components/article';
+import Author from './components/author';
+import Reader from './components/reader';
+import Home from './components/home';
```

Comment:
filename should match the export name. https://github.com/airbnb/javascript#naming--filename-matches-export

Not necessary, but let's keep consistent with this style guidance.
https://github.com/airbnb/javascript/blob/master/react/README.md